qa/standalone/test_ceph_daemon.sh

```diff
@@ -30,6 +30,10 @@ $SUDO $CEPH_DAEMON --image $IMAGE version | grep 'ceph version'
 # try force docker; this won't work if docker isn't installed
 which docker && ( $SUDO $CEPH_DAEMON --docker version | grep 'ceph version' )
 
+## test shell before bootstrap, when crash dir isn't (yet) present on this host
+$SUDO $CEPH_DAEMON shell -- ceph -v | grep 'ceph version'
```

Comment:
```
2019-10-26T17:29:42.004 INFO:tasks.workunit.client.0.mira109.stderr:+ sudo /usr/sbin/ceph-daemon shell -- ceph -v
2019-10-26T17:29:42.004 INFO:tasks.workunit.client.0.mira109.stderr:+ grep 'ceph version'
2019-10-26T17:29:42.149 INFO:tasks.workunit.client.0.mira109.stderr:the input device is not a TTY
2019-10-26T17:29:42.171 DEBUG:teuthology.orchestra.run:got remote process result: 1
```

@liewegas this test fails. /a//kchai-2019-10-26_15:51:16-rados-wip-kefu-testing-2019-10-26-1857-distro-basic-mira/4447747

Reply:
https://tracker.ceph.com/issues/42499